buildSrc/src/main/java/org/elasticsearch/gradle/DistributionDownloadPlugin.java

```diff
@@ -66,6 +67,9 @@
 
     @Override
     public void apply(Project project) {
+        // this is needed for isInternal
```

Comment:
Can we be more specific here and reference `BuildParams`?

Reply:
I left it generic because this will be backported to 7.5 where it is still an ext property.